jvm/src/main/scala/scalajson/ast/JValue.scala

```diff
@@ -64,7 +64,7 @@ object JNumber {
 
   def apply(value: Integer): JNumber = JNumber(value.toString)
 
-  def apply(value: Array[Char]): JNumber = JNumber(value.toString)
+  def apply(value: Array[Char]): JNumber = JNumber(value.mkString)
```

Comment:
`mkString` is a bit slow compared to using the String constructor directly, but that's how it's implemented elsewhere. I'm going for consistency in this PR, but I've put up a 3x faster version that also includes this fix here: https://github.com/mdedetrich/scalajson/pull/26

Please close this PR if you decide to merge the other one.